plugin/plugin.go

```diff
@@ -0,0 +1,86 @@
+package plugin
+
+import (
+	context "context"
```

Comment:
Does the import need to be named explicitly here?

Reply:
No idea why it did that. Some auto-insertion thing. Will remoev.